src/platform/packages/shared/kbn-esql-validation-autocomplete/src/definitions/commands.ts

```diff
@@ -243,6 +245,7 @@ export const commandDefinitions: Array<CommandDefinition<any>> = [
     modes: [],
     // Reusing the same validation logic as stats command
     validate: statsValidator,
+    suggest: suggestForStats,
```

Comment:
Is this added to add autocomplete support to inlinestats? If yes it doesnt seem to work as expected

<img width="785" alt="image" src="https://github.com/user-attachments/assets/31d295bb-3d03-4f2f-9f60-0cf4bff9fd64" />

Reply:
Thanks... actually what is the status of `inlinestats`? I don't see it in our docs...

Reply:
Inlinestats is paused atm in favor of joins. Let's remove it the suggest here and deal with it when inlinestats is back in the roadmap.

Reply:
Done in https://github.com/elastic/kibana/pull/212996/commits/0e6fc40f81b71237e8276afba43232d586c90940